lib/server.js

```diff
@@ -57,6 +57,22 @@ function create(log, error, config, routes, db) {
     }
   }
 
+  function makeVerifiedCredentialFn (dbGetFn) {
+    return function (id, cb) {
+      log.trace({ op: 'DB.getToken', id: id })
+      if (!HEX_STRING.test(id)) {
+        return process.nextTick(cb.bind(null, null, null)) // not found
+      }
+      dbGetFn(Buffer(id, 'hex'))
+        .done(
+          function (token) {
+            return cb(null, token)
+          },
+          cb
+        )
+    }
+  }
```

Comment:
It's not obvious why we need a separate `makeVerifiedCredentialsFn` rather than re-using the almost identical `makeCredentialsFn` - is this a holder from when the strategy for explicitly checking the verification status rather than returning it?